### PR TITLE
Delete analyze.yml

### PR DESCRIPTION
**Pull request by rotu**
This creates noise on each pull request. It also makes the Github search less useful, since even trivial changes (e.g. renaming a file) can alter the built sizes of various component (see, e.g. https://github.com/search?q=repo%3Amaplibre%2Fmaplibre-gl-js+earcut&type=pullrequests)

### CODE REVIEW

**Comment by HarelM** [comment]
I don't know how I feel about this honestly, @birkskyum , @wipfli what do you guys think?
I tend to agree I usually don't care for its results...

**Comment by birkskyum** [comment]
I didn't know that it had these implications to searching.

I always check it to make sure the bundle size doesn't increase, but can we do that in another way?

**Comment by wipfli** [comment]
I don't worry about the extra message in the pull request thread. So this alone would not justify for me removing the analyze workflow. But you are right that it makes searching hard. Maybe you can delete the "view details"?

**Comment by rotu** [comment]
Maybe we could just add a post-build unit test that the resulting files are under some threshold.

**Comment by birkskyum** [comment]
I'd like to understand the issue regarding search - how does this plugin make searching less useful

**Comment by birkskyum** [comment]
Also the link in the description seems to be broken, or maybe I don't know what to look for

**Comment by rotu** [comment]
> Also the link in the description seems to be broken, or maybe I don't know what to look for

That link serves to demonstrate the search issue. The link is a search for "earcut" but note the results don't actually pertain to the "earcut" functionality - you'll notice that the only place "earcut" is mentioned is in the (initially collapsed) details of the bundle size report.

**Comment by birkskyum** [comment]
That is annoying, but wouldn't the ear it text also not be found if we disabled the Analyse?

**Comment by rotu** [comment]
> That is annoying, but wouldn't the earcut text also not be found if we disabled the Analyse? How does it improve the search to remove it?

The analyze workflow creates (via the `marocchino/sticky-pull-request-comment` action) a comment on each pull request. Removing the workflow prevents that comment from being auto-generated.

I feel like PR checks should have clear action items. Otherwise they’re just distracting (“what is this information and what am I supposed to do with it?”)

File: test/build/min.test.ts [hunk 1]
```diff
@@ -33,4 +33,12 @@ describe('test min build', () => {
         }
     });
 
+    test('bundle size stays the same', async () => {
+        const bytes = (await fs.promises.stat('dist/maplibre-gl.js')).size;
+        const alpha = 0.1;
```
Comment (HarelM) [inline test/build/min.test.ts:38]:
```suggestion
        const alpha = 0.05;
```

**Review by HarelM — APPROVED** [review]

**Pull request merged**